lib/Controller/WebAssetController.php

```diff
@@ -62,7 +61,7 @@ public function get(): Response {
 
 		try {
 			return new DataDisplayResponse(\file_get_contents($filePath), Http::STATUS_OK, [
-				'Content-Type' => $this->getMimeType($filePath),
+				'Content-Type' => 'text/javascript',
```

Comment:
@xoxys cc

Reply:
looks little fishy tbh

Reply:
discussed in zoom, I'm fine with it for this super simple case

Reply:
and for the record: in [web](https://github.com/owncloud/web/blob/acfc3165b56fadd16db85e7aaefaea8cb1d0e98d/packages/web-integration-oc10/lib/Controller/FilesController.php#L61) we inject a mimetype detector that comes along from within oc10. I decided to not use that because I don't expect that we'll ever serve another file than this single javascript file.